planex/cmd/clone.py

```diff
@@ -42,6 +42,8 @@ def parse_args_or_exit(argv=None):
                            "link the sources and the patchqueue and use "
                            "guilt to apply all the additional patches "
                            "[experimental]")
+    parser.add_argument("--output", "-o", default=".",
```

Comment:
I think an output filename rather than a directory would be clearer.

Reply:
I was going for a fixed filename since I used a fixed one in the build scripts admittedly.

Reply:
As the caller consumes the file it better for that to define the entire filename.